x-pack/plugins/uptime/server/lib/adapters/telemetry/kibana_telemetry_adapter.ts

```diff
@@ -132,7 +130,7 @@ export class KibanaTelemetryAdapter {
   }
 
   public static async countNoOfUniqueMonitorAndLocations(
-    callCluster: ILegacyScopedClusterClient['callAsCurrentUser'] | UptimeESClient,
+    callCluster: ElasticsearchClient | UptimeESClient,
```

Comment:
this will now only be `UptimeESClient`

Reply:
Thanks for catching that! [Done](https://github.com/elastic/kibana/pull/86718/commits/f63fe37a28464ef58583c71a0bc68d2cb26b0969#diff-b14707a96d135f28f2326ccb972bcf2b15514928e2c73d9627e3ea79c29001d8R128)